### ISSUE TITLE
petition system to reach leadership, members

### ISSUE THREAD

**Issue by dckc** [rchain/bounties#380]
In the "Cooperative Governance" session today led by @drbloom, there was considerable support for a petition system that serve as a threshold for
  - getting an item on the agenda of [Greg's community debrief](https://github.com/rchain/bounties/wiki/Weekly-Debrief-Index)
  - getting a message pushed out to all members via email (or other means of reaching their smartphones)

_In the same session, there was support for a Membership Working Group for orientation and such, and this task may / should fall to that group. But until that group has a work queue that we can all see, I'm recording it here._

cc @patrick727

**Comment by dckc** [comment]
Peter suggested [loomio](https://www.loomio.org/) based on first hand experience in Resonate; I spent a little time checking it out and I'm interested to try it. It's open source and we can self-host it; Jeremy said he's willing to stand it up.
 
@drbloom I was a little surprised that you didn't mention that you had actually set up a petition system in discord. I guess it was obvious that it wasn't sufficiently usable or whatever.

**Comment by ICA3DaR5** [comment]
May I suggest [Bountysource](https://www.bountysource.com/) ?

**Comment by dckc** [comment]
I'm kicking the tires on loomio; anybody is welcome to join: [RChain Loomio Experiment](https://www.loomio.org/invitations/33d924be03c1cd7eddc6).

@ICA3DaR5 I hope to check those things out.

**Comment by dckc** [comment]
https://wordpress.org/plugins/yop-poll/ looks nice. (hat tip: @jeremybusk)

**Comment by dckc** [comment]
>  Ian [@drbloom] is working on creating a membership listserv to help with the issue of alerting all members. -- [Henderson Feb 28](https://lists.w3.org/Archives/Public/www-archive/2018Feb/0012.html)

**Comment by drbloom** [comment]
@dckc No I am not working on a listserv to help with the issue of alerting all members.
Please remove me from https://lists.w3.org/Archives/Public/www-archive/2018Feb/

**Comment by dckc** [comment]
Henderson requested an edit to www-archive and I endorsed it. I haven't heard back yet.

**Comment by dckc** [comment]
The thread around Henderson Feb 28 has been removed from the W3C archive.

**Comment by dckc** [comment]
@jeremybusk and @ian-bloom and I were talking last night and we're leaning toward self-hosting a loomio instance. For one thing, it opens up the possibility of authenticating coop members via OAuth integration with discord.

We didn't talk much about migrating the content from the existing loomio experiment; I doubt it's cost-effective.

Meanwhile, the existing discord `#polling-station` channel is in place.

**Comment by kitblake** [comment]
For issue #446 I created a "Ranked Choice" poll in Loomio. 

Creating the poll was easy, seamless and intuitive. I really like Loomio! When I use their software it feels 'fully formed', meaning it's been thought thru, put thru its paces, optimized, and it harbors a wealth of invisible decisions.

The Coop would really benefit from adopting all they've done.

**Comment by dckc** [comment]
Yes, I agree. I think it would take us 9 to 24 months  to configure wordpress to do 80% of the relevant loomio features.

And the loomio docs about making decisions in a community are really great. They line up very well with my 15 years of experience at W3C + 5 years running collaborative muliti-site heath research projects.

**Comment by ian-bloom** [comment]
I'm a fan and _former_ user of Loomio. I agree that it is great tool and they are structured as a Cooperative.

The difficulties I see are:

1. [Pricing](https://www.loomio.org/pricing), if we were to include all of our ~1100 members, would be very high. (Self-hosting is also an option)
3. Additional admin and maintenance
4. Learning curve, and adoption pains of shepherding members onto _yet another_ platform.This is fun for us geeks, but can be quite painful for others.
2. Too many channels
  > “Too many channels. Too many messages. Too much noise. Too much guilt… The world sends me tweets, direct messages, texts, chats with me on skype, sends me Facebook emails (!) and actual mail and also calls me… Responding on all these channels is a full time job, extremely distracting and exhausting. I feel constantly behind.”
--- Tina Roth Eisenberg (excerpted from [Swiss Miss](http://www.swiss-miss.com/2011/03/communication-crisis.html))

**Comment by dckc** [comment]
It's good to talk through these issues; they're not trivial. But loomio still looks like an effective way forward to me; I don't see much in the way of new concerns:

1. Pricing seems manageable. It's not like all 1100 members are going to participate on day 1. It sure looks to me like self-hosting would cost us more than what they would charge us, but I'm content to cross that bridge when we come to it.
2. Additional admin over what? Yes, it's more than not doing it. I doubt it's more administration than doing it ourselves in a wordpress-based system with 80% of the relevant features. I [asked for loomio admin volunteers](https://www.loomio.org/p/fa2DXTko/who-wants-to-help-administer-this-loomio-instance-) and got about 4 or 5 within a week, by the way.
3. Again: _yet another_ platform / channel vs what?
   - Yes, it's yet another compared to what we have now, which is discord.
   - If discord were sufficient
      - people at the the gov forum might have known about it and been content with it
      - We wouldn't see ~15 people engaging pretty actively in an experimental loomio instance

**Comment by kitblake** [comment]
@ian-bloom, responding to your 4 points above. 

1. Pricing: I think the Co-op can afford it.  \
We'll never have ~1100 members actually using it.  \
Plus there's a non-profit deal, and I bet that if we negotiate with the Loomio people, we can get an even better sister-cooperative deal. 

About points 2, 3, and 4: If we do nothing, then they won't apply. But is we do something, then whatever that something is, they'll all apply. So we might as well go with Loomio and have beautiful tools.

**Comment by kitblake** [comment]
@patrick727 Hey, you talked with the Loomio founders.... think they'd be open to a special, sister-cooperative, type deal? 

Plus, would they accept payments in crypto? Like, RHOC?

**Comment by dckc** [comment]
@kennyrowe and @glenbraun and I met yesterday to talk about coop processesin Rholang. This "manageable inbox" process is high on Kenny's wish list.

Tangentially related, Glenn suggested a personal rnode per member. I didn't understand the motivation at first, but it's basically: privacy.

Today Glenn and I brainstormed a bit about how to do hybrid messaging where repuation is kept on the public chain and the content of messages goes point-to-point between personal nodes. I think I'm starting to get it.
